scripts/nb-tester/qiskit_docs_notebook_tester/__init__.py

```diff
@@ -10,10 +10,14 @@
 # notice, and modified files need to carry a notice indicating that they have
 # been altered from the originals.
 
+from __future__ import annotations
+
+
 import argparse
 import asyncio
 import sys
 import textwrap
+import tomllib
```

Comment:
This will cause a `ModuleNotFoundError`, which is a subclass of `ImportError`. Instead, the Python idiom is:

```python
try:
    import tomllib
except ImportError:
    import tomli as tomllib
```

Fun fact: I first got involved in Python dev tooling by doing Python 2 -> 3 migrations 🐍

Reply:
Is there any advantage to the try/except over just using `tomli` all the time?

Reply:
No, it's simpler to always use `tomli`. But I wouldn't rename it to be `tomllib` because that's confusing - they're not 1:1 libraries.